content/24.vaccines-novel.md

```diff
@@ -385,44 +376,59 @@ Ultimately, primed T follicular helper cells can stimulate germinal center B cel
 The second mRNA vaccine, mRNA-1273 developed by ModernaTX, is comprised by a conventional LNP-encapsulated RNA encoding a full-length prefusion stabilized S protein for SARS-CoV-2 [@clinicaltrials:NCT04283461].
-The vaccine candidates developed against SARS-CoV-2 using mRNA vectors utilize similar principles and technologies, although there are slight differences in implementation among candidates such as the formulation of the platform and the specific components of the spike protein encapsulated (e.g., the full Spike protein vs. the RBD alone) [@doi:10.3389/fphar.2020.00937].
+The vaccine candidates developed against SARS-CoV-2 using mRNA vectors utilize similar principles and technologies, although there are slight differences in implementation among candidates such as the formulation of the platform and the specific components of the Spike protein encapsulated (e.g., the full Spike protein vs. the RBD alone) [@doi:10.3389/fphar.2020.00937].
 As of {{owid_most_recent_date}}, {{owid_RNA_count}} mRNA vaccines are available in {{owid_RNA_countries}} countries (Figure @fig:mRNA-distrib).
 
-<!-- A general comment is that it can be confusing why the number of vaccines in the tables (approve) and figures (available) are different. -->
-
 ![
 **Worldwide availability of vaccines developed using mRNA.**
 This figure reflects the number of vaccines based on mRNA technology that were available in each country as of {{owid_most_recent_date}}.
 These data are retrieved from Our World in Data [@doi:10.1038/s41562-021-01122-8] and plotted using geopandas [@doi:10.5281/zenodo.5573592].
 See https://greenelab.github.io/covid19-review/ for the most recent version of this figure, which is updated daily.
+Note that this figure draws from a different data source than Table @tbl:approved-RNA and does not necessarily include data for every vaccine developed within this category.
 ]({{owid_RNA_map}} "Availability of mRNA vaccines"){#fig:mRNA-distrib secno=1}
 
-**Trial Safety and Immunogenicity:**
+The rapid and simultaneous development of these vaccines was met with some controversy related to intellectual property (IP).
+First, the National Institutes of Health (NIH) and Moderna became involved in a patent dispute, after researchers at the NIH argued they were unfairly excluded from some patents filed based on their IP after they generated the stabilized modRNA sequence used in the vaccine [@doi:10.1038/d41586-021-03535-x].
+Ultimately, in late 2021, Moderna backed down on the patent application [@url:https://www.washingtonpost.com/health/2021/12/17/moderna-vaccine-patent-dispute-nih].
+However, in August 2022, the company filed their own suit against Pfizer/BioNTech over IP related to to the modRNA used in the latter's COVID-19 vaccine [@url:https://www.washingtonpost.com/health/2021/12/17/moderna-vaccine-patent-dispute-nih; @url:https://www.cnn.com/2022/08/26/health/moderna-pfizer-mrna-patent-lawsuit/index.html].
+The outcome of this suit remains to be seen.
```

Comment:
Too bad we never finished work on the very first manubot issue `https://github.com/manubot/manubot/issues/1`: support for citing legal cases!